change-notes/1.20/analysis-python.md

```diff
@@ -23,4 +23,4 @@
 
  ## Changes to QL libraries
 
- * *Series of bullet points*
+ * Added support for the `dill` pickle library.
```

Comment:
Thanks for the change note and for mentioning me. LGTM.

Would you mind mentioning @felicity-semmle for future PRs to avoid spamming the rest of the doc team?

Reply:
Of course! I will do that from now on.